Gooey/Sources/UIViewController.swift

```diff
@@ -20,6 +20,15 @@ public extension GooeyNamespace where Base: UIViewController {
         return base.view.safeAreaLayoutGuide.bottomAnchor
     }
 
+    /// Returns the bottom-most anchor for visible content.
+    ///
+    /// On the iPhone X, this anchor falls behind the virtual home button for view controllers with
+    // bounds equal the window's bounds.
+    var bottomBleedAnchor: NSLayoutYAxisAnchor {
+        guard !DeviceCheck.isModelX else { return base.view.bottomAnchor }
```

Comment:
![ModelX](https://cnet4.cbsistatic.com/img/3-14WgVgrMGZ_pKcAqsefdLScJc=/830x467/2016/04/28/5a91e8fc-14bc-4c22-abe4-b990073328d2/2016-tesla-model-s-19-of-43.jpg)